pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/readers/forward/FixedByteChunkSVForwardIndexReader.java

```diff
@@ -91,4 +264,8 @@ public double getDouble(int docId, ChunkReaderContext context) {
       return _rawData.getDouble(docId * Double.BYTES);
     }
   }
+
+  private boolean isContiguousRange(int[] docIds, int length) {
```

Comment:
nit: it'd be helpful to comment a bit when the range can be contiguous and when not.

Reply:
Why? That doesn't seem like a responsibility of this class, does it?

Reply:
Think I'm lack of some context to understand this new check. Not necessarily a comment to the code. Would appreciate it if you could shed some light in the conversion here. Just looking at this check method, I assume it'd require the docIds to be unique and sorted in asc. So when that happens and when not. This may help me understand when the optimization can kick in.

Reply:
OK, got it. However, I think that kind of info would be better off in a readme about the how the query and storage layers interact. I can do that (because understanding this area of the code has been hard for me) but not in this PR. I don't think the comment belongs here, and would be superfluous after reading a decent readme.